### PR TITLE
Rename module to spryker-smart-inbox-connector

### PR DESCRIPTION
**Pull request by flagbird**
This PR also needs to be tested in a running Spryker instance.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "one-and-one/one-and-one-mail-connector",
+  "name": "flagbit/spryker-smart-inbox-connector",
```
Comment (amansilla) [inline composer.json:2]: A change in composer package name will break the spryker twig templating. The module of the namespace should be changed as well for this to be ok.

Comment (flagbird) [reply composer.json:2]: Technically it is possible to use this name for the composer package. Would this be sufficient to guarantee the runtime of the lib?

File: src/Flagbit/Zed/SprykerSmartInboxConnector/Communication/Controller/IndexController.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ */
+class IndexController extends AbstractController
+{
+    /**
+     * @return array
+     */
+    public function indexAction()
+    {
+        return [
+            'foo' => 'hello worlkd',
```
Comment (wapec) [inline src/Flagbit/Zed/SprykerSmartInboxConnector/Communication/Controller/IndexController.php:19]: worlkd?

Comment (amansilla) [reply src/Flagbit/Zed/SprykerSmartInboxConnector/Communication/Controller/IndexController.php:19]: :D in fact I guess we don't need this controller, so the whole class can be removed.

**Comment by amansilla** [comment]
Can anyone confirm this was tested on a Spryker running project and verified the email is generated as expected? Otherwise I will test it locally next Wednesday 12.05.

**Review by wapec — DISMISSED** [review]
lgtm

**Comment by flagbird** [comment]
@wapec Was it testable with https://github.com/flagbit/spryker-b2c-demo-shop/pull/1 ? I had issues to install it while being an separated branch because of the renaming.

**Comment by amansilla** [comment]
Not only installing is enough. The whole functionality should be tested since because due the renaming the most fragile part is the twig template ñame resolution. That only happens when the email is sent. ;)

________________________________

**Comment by flagbird** [comment]
> The whole functionality should be tested since because due the renaming the most fragile part is the twig template ñame resolution.

This will need an integration test in the future to cover the fragile properties.

**Comment by amansilla** [comment]
@flagbird you're right about the issues with composer install. I've tried to install the package and investigated how composer behaves and investigated for similar issues (see https://github.com/composer/packagist/issues/47). Composer will only take into account the package name defined in the `main` branch for the repo. So being able to test this we should merge it in the `main` branch what is not nice, but if we want to rename it there are no more options. Since local testing is not possible I'd suggest making just the code review and merge for later testing once merged. The new version of the package won't be available until we tag a new version. I know it's not nice but I can't see any further options. Hm yes, not renaming anything :D. @flagbird thoughts?

**Comment by flagbird** [comment]
> So being able to test this we should merge it in the main branch what is not nice...

@amansilla Yes it's unfavourable, but a merge into `main` is not the same as a release, unlike in other projects. That's why a merge into `main` should be okay as long as we don't release a buggy version.

**Review by amansilla — APPROVED** [review]
As agreed with @flagbird since testing on the feature branch is impossible, we'll merge this for later testing on the `main` branch. It shouldn't be an issue as far as we don't tag a new version.

**Pull request merged**